factory/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""All necessary class initializations."""
+from db.facade import DBFacade
+from db.dynamodb import DynamoDB
+from command.core import Core
+from slackclient import SlackClient
+import os
+
+
+def make_core():
+    """
+    Initialize and returns a :class:`command.core.Core` object.
+
+    Requires that environmental variable ``SLACK_API_TOKEN`` to be set. Please
+    set it to be the slack API token.
+
+    :return: a new ``Core`` object, freshly initialized
+    """
+    slack_token = os.environ['SLACK_API_TOKEN']
```

Comment:
We may want to consider starting to put some of these into a config file, instead of stuffing them all in env vars - that wouldn't have to be in this PR (unless you want to do it...) but I think this would be the place to read everything from the config.